### PR TITLE
OKTA-158311_move_whitelisting

### PR DESCRIPTION
**Pull request by eklein-okta**
## Description:
- Moved info Okta EL to Creating Token with Groups Claim How To
### Resolves:
* [Issue #X](#X)

* [OKTA-158311](https://oktainc.atlassian.net/browse/OKTA-158311)

### CODE REVIEW

File: _source/_docs/how-to/creating-token-with-groups-claim.md [hunk 1]
```diff
@@ -18,16 +18,26 @@ Perform the following two tasks before you start for either Okta Authorization S
...
+
+Now, use the instructions for your chosen authorization server to create a groups claim, assign a group whitelist to your client app, and configure a groups claim that references a whitelist:
 
 * [Create a Token with Groups Claim and Okta Authorization Server](#create-a-token-with-groups-claim-okta-authorization-server)
 * [Create a Token with Groups Claim and Custom Authorization Server](#create-a-token-with-groups-claim-custom-authorization-server)
 
-### Create a Token with Groups Claim (Okta Authorization Server)
+### Use Group Functions for Static and Dynamic Whitelists
+
+You can use the [Okta Expression Language Group Functions](../../reference/okta_expression_language#group-functions)  to use static and dynamic whitelists.
```
Comment (robertjd) [inline _source/_docs/how-to/creating-token-with-groups-claim.md]: Hi @eklein-okta , these links should be written as `/reference/etc` without the leading docs.  Once you update this the tests should pass, thankks!

Comment (mystiberry-okta) [reply _source/_docs/how-to/creating-token-with-groups-claim.md]: I forgot to mention that! links to other content in developer.okta.com use abbreviated references, whatever comes after the http://developer.okta.com/docs usually. Not relative references anymore.

File: _source/_docs/how-to/creating-token-with-groups-claim.md [hunk 2]
```diff
@@ -18,16 +18,26 @@ Perform the following two tasks before you start for either Okta Authorization S
  * Create an OAuth 2.0 or OpenID Connect client with the [Apps API](/docs/api/resources/apps#request-example-8). In the instruction examples, the client ID is `0oabskvc6442nkvQO0h7`.
  * Create the groups that you wish to configure in the groups claim. In the instruction examples, we're configuring the `WestCoastDivision` group, and the group ID is `00gbso71miOMjxHRW0h7`.
 
-Now use the instructions for your chosen authorization server to create a groups claim, assign a group whitelist to your client app, and configure a groups claim that references the whitelist:
+### Use the Authorization Server with a Static Whitelist 
+
```
Comment (mystiberry-okta) [inline _source/_docs/how-to/creating-token-with-groups-claim.md]: The user goal isn't to use the auth server, it's to create a groups claim using a static whitelist.

File: _source/_docs/how-to/creating-token-with-groups-claim.md [hunk 3]
```diff
@@ -18,16 +18,26 @@ Perform the following two tasks before you start for either Okta Authorization S
...
+Now, use the instructions for your chosen authorization server to create a groups claim, assign a group whitelist to your client app, and configure a groups claim that references a whitelist:
 
 * [Create a Token with Groups Claim and Okta Authorization Server](#create-a-token-with-groups-claim-okta-authorization-server)
 * [Create a Token with Groups Claim and Custom Authorization Server](#create-a-token-with-groups-claim-custom-authorization-server)
 
-### Create a Token with Groups Claim (Okta Authorization Server)
+### Use Group Functions for Static and Dynamic Whitelists
+
```
Comment (mystiberry-okta) [inline _source/_docs/how-to/creating-token-with-groups-claim.md]: I thought we were going to split it up like this:

Creating groups claims with a static whitelist
Creating groups claims with a dynamic whitelist

**Comment by eklein-okta** [comment]
Changes made...

**Comment by eklein-okta** [comment]
OKTA-158311 completed!

**Pull request merged**